website/eacr/templates/500.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
And I think they should extend `homepage.html`, because `base.html` is very barebone so we can use it for other subsites as well.